WORKSPACE

```diff
@@ -0,0 +1,28 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
```

Comment:
Seems like a lot of go_repository entries are missing. Is that no longer required by rules_go?